Source/Widgets/CesiumWidget/CesiumWidget.js

```diff
@@ -692,6 +696,11 @@ define([
 
         configureCanvasSize(this);
         configureCameraFrustum(this);
+
+        var scene = this._scene;
+        if (scene.requestRenderMode) {
```

Comment:
Is there any harm in calling `scene.requestRender()` if request render mode is off?  I would get rid of this if statement.